internal/chezmoi/findexecutable_darwin_test.go

```diff
@@ -0,0 +1,90 @@
+//go:build darwin
+
+package chezmoi
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFindExecutableOneNeedle(t *testing.T) {
+	tests := []struct {
+		file     string
+		paths    []string
+		expected string
+	}{
+		{
+			file:     "sh",
+			paths:    []string{"/usr/bin", "/bin"},
+			expected: "/bin/sh",
+		},
+		{
+			file:     "sh",
+			paths:    []string{"/bin", "/usr/bin"},
+			expected: "/bin/sh",
+		},
+		{
+			file:     "chezmoish",
+			paths:    []string{"/bin", "/usr/bin"},
+			expected: "",
+		},
+	}
+
+	for _, test := range tests {
+		name := fmt.Sprintf("FindExecutable %v in %#v as %v", test.file, test.paths, test.expected)
+		t.Run(name, func(t *testing.T) {
+			actual, err := FindExecutable(test.file, test.paths)
+			if err != nil {
+				t.Errorf("FindExecutable() error = %v, expected = nil", err)
+			}
+
+			if actual != test.expected {
+				t.Errorf("FindExecutable() actual = %v, expected = %v", actual, test.expected)
+			}
```

Comment:
Instead of writing these assertions out in Go, the rest of the code uses github.com/alecthomas/assert/v2 which means this block can be written as:

```go
actual, err := FindExecutable(test.file, test.paths)
assert.NoError(t, err)
assert.Equal(t, test.expected, actual)
```

This same comment applies to the other tests in this PR.

Reply:
Done. Let’s see what the automated build says for both of these changes.